projects/cashmere/src/lib/date-range/calendar-wrapper/calendar-wrapper.component.scss

```diff
@@ -3,6 +3,6 @@
     height: 70px;
 }
 
-.form-container {
+.hc-calendar-wrapper > .form-container {
     padding-left: 13px;
```

Comment:
`.form-container` was actually colliding with a couple other examples on the demo site.  So I moved it into specific namespace for the component to make sure this wouldn't happen in any apps.